.circleci/config.yml

```diff
@@ -311,6 +315,23 @@ commands:
       - run:
           command: ./.circleci/report-coverage.sh << parameters.list >>
 
+  run-playwright-tests:
+    parameters:
+      project:
+        type: string
+    steps:
+      - run:
+          name: Running Playwright tests
+          # Supports 'Re-run failed tests only'. See this for more info: https://circleci.com/docs/rerun-failed-tests-only/
+          command: |
+            cd packages/functional-tests
+            TEST_FILES=$(circleci tests glob "tests/**/*.spec.ts")
+            echo $TEST_FILES | circleci tests run --command="xargs yarn playwright test --project=<< parameters.project >>" --verbose --split-by=timings
+          environment:
+            NODE_OPTIONS: --dns-result-order=ipv4first
+            JEST_JUNIT_OUTPUT_DIR: ./artifacts/tests
+            JEST_JUNIT_ADD_FILE_ATTRIBUTE: true
+
```

Comment:
Q: Is this related to Redis changes, or a hop-on from the other PR?

Reply:
Oh, good catch. That must have gotten mixed in somehow.

Reply:
Turns out the branch wasn't based on main. Rebasing fixed this.